### ISSUE TITLE
Animal Dungeon - Lava Cow Boss Fight

### ISSUE THREAD

**Issue by MineCaftatron85** [Wynncraft/Issues#3853]
When fighting Lava Cow in the Animal Dungeon, I knocked him through invisible walls and out of bounds.

In the arena, there are invisible walls preventing the player from moving away from the boss fight area.
So, being an Archer, I used the "Arrow Storm" spell and knocked back the Lava Cow through the invisible walls.

His projectiles were blocked by the invisible walls and so were mine.
I tried to move to the far side of the arena but Lava Cow didn't teleport to me.

This could potentially stall the boss fight, preventing the player from finishing the fight.
I only defeated him by using my "Bomb Arrow" ability to hit him through the invisible walls.

**Comment by Trixter313** [comment]
Duplicate of https://github.com/Wynncraft/Issues/issues/3191

Please direct any future comments to the original thread. Thanks!
